scripts/CEE/get-kafka-instance-state/metadata.yaml

```diff
@@ -83,4 +83,10 @@ envs:
   - key: "since_time"
     description: "This is being passed to use the --since-time flag used by oc adm inspect. Timestamp is in RFC 3339 format. Only one of since-time/since may be used"
     optional: true
+  - key: "kafka_max_transaction_timeout"
+    description: "Max transaction timeout in minutes to use when querying for hanging transactions"
+    optional: true
+  - key: "kafka_num_brokers"
```

Comment:
is there a use-case for running the command against fewer than the number of brokers the instance has?  If no, I think the script should work out the number of brokers that exist, and run it against all.

Reply:
> is there a use-case for running the command against fewer than the number of brokers the instance has? If no, I think the script should work out the number of brokers that exist, and run it against all.

Good idea, thanks. I'll try that.

> Also I notice that the [--broker-id BROKER_ID] is marked as optional, do we actually need to pass it at all?

The `find-hanging` command requires either `--topic` or `--broker-id` to limit the scope of the search. It seems easier to pass the broker id.